packages/node/src/notifier.js

```diff
@@ -79,7 +79,7 @@ Object.keys(Client.prototype).forEach((m) => {
   if (/^_/.test(m)) return
   Bugsnag[m] = function () {
     // if we are in an async context, use the client from that context
-    const client = Bugsnag._client._clientContext.getStore() || Bugsnag._client
+    const client = Bugsnag._client?._clientContext?.getStore() || Bugsnag._client
```

Comment:
if we're still supporting IE11 then we'll have to drop the optional chaining I'm afraid

@tomlongridge does v8 drop IE11 support?

Reply:
this code only runs on node

Reply:
though the node e2e tests are failing so maybe it was only added in node 20. will revert to the tried and tested ways then